lib/projections/etmerc.js

```diff
@@ -0,0 +1,161 @@
+// Heavily based on this etmerc projection implementation
+// https://github.com/mbloch/mapshaper-proj/blob/master/src/projections/etmerc.js
+
+var sinh = require('../common/sinh');
+var hypot = require('../common/hypot');
+var asinhy = require('../common/asinhy');
+var gatg = require('../common/gatg');
+var clens = require('../common/clens');
+var clens_cmplx = require('../common/clens_cmplx');
+var adjust_lon = require('../common/adjust_lon');
+
+var cgb = [];
+var cbg = [];
+var utg = [];
+var gtu = [];
+var Qn;
+var Zb;
+
+exports.init = function() {
+  if (this.es <= 0) {
+    return;
+  }
+
+  this.x0 = this.x0 !== undefined ? this.x0 : 0;
+  this.y0 = this.y0 !== undefined ? this.y0 : 0;
+  this.long0 = this.long0 !== undefined ? this.long0 : 0;
+  this.lat0 = this.lat0 !== undefined ? this.lat0 : 0;
+
+  var f = this.es / (1 + Math.sqrt(1 - this.es));
+  var n = f / (2 - f);
+  var np = n;
+
+  cgb[0] = n * (2 + n * (-2 / 3 + n * (-2 + n * (116 / 45 + n * (26 / 45 + n * (-2854 / 675 ))))));
```

Comment:
Yeah, looks like they should be local to the object, since they might differ for each new proj4 instance with a different proj4 string if we create two separate instances, right? I will test that when I will get to my PC and fix it.